### PR TITLE
docs: add LSP link + Github icon on top nav bar

### PR DESCRIPTION
**Pull request by CJ42**
# What does this PR introduce?

- [x] add link to LSP repo **(open in new tab)** on top nav bar 
- Change "Github" text link by **github icon**

_Before:_
Screenshot 2022-01-19 at 13 33 27

_After:_
Screenshot 2022-01-19 at 13 33 21

Signed-off-by: CJ42 <cavallerajean@gmail.com>

### CODE REVIEW

**Comment by Hugoo** [comment]
It looks like the GitHub logo is a bit up?

image

**Comment by CJ42** [comment]
@Hugoo fixed 👌 
Screenshot 2022-01-19 at 13 40 04

**Comment by CJ42** [comment]
> It looks like the GitHub logo is a bit up?
> 
> image

**Comment by CJ42** [comment]
> > It looks like the GitHub logo is a bit up?
> > image

@Hugoo fixed 👌 
Screenshot 2022-01-19 at 13 40 04

File: src/css/custom.css [hunk 1]
```diff
@@ -42,3 +42,39 @@ html[data-theme='dark'] {
...
+    no-repeat;
+}
+
+html[data-theme='dark'] .header-github-link:before {
+  background: url("data:image/svg+xml,%3Csvg viewBox='0 0 24 24' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath fill='white' d='M12 .297c-6.63 0-12 5.373-12 12 0 5.303 3.438 9.8 8.205 11.385.6.113.82-.258.82-.577 0-.285-.01-1.04-.015-2.04-3.338.724-4.042-1.61-4.042-1.61C4.422 18.07 3.633 17.7 3.633 17.7c-1.087-.744.084-.729.084-.729 1.205.084 1.838 1.236 1.838 1.236 1.07 1.835 2.809 1.305 3.495.998.108-.776.417-1.305.76-1.605-2.665-.3-5.466-1.332-5.466-5.93 0-1.31.465-2.38 1.235-3.22-.135-.303-.54-1.523.105-3.176 0 0 1.005-.322 3.3 1.23.96-.267 1.98-.399 3-.405 1.02.006 2.04.138 3 .405 2.28-1.552 3.285-1.23 3.285-1.23.645 1.653.24 2.873.12 3.176.765.84 1.23 1.91 1.23 3.22 0 4.61-2.805 5.625-5.475 5.92.42.36.81 1.096.81 2.22 0 1.606-.015 2.896-.015 3.286 0 .315.21.69.825.57C20.565 22.092 24 17.592 24 12.297c0-6.627-5.373-12-12-12'/%3E%3C/svg%3E")
```
Comment (Hugoo) [inline src/css/custom.css:61]: I recommend to not use data image inside the css but to use a .svg image instead (either stored in this repo as static asset or on a CDN).

Reasons:

1. the image can be cached separately (if you change one letter in the css file, the clients won't have to download the .svg again, only the new .css)
2. makes the css less ugly
3. When browsing the repo, it is easier to see how the image looks like
4. The image can be re used elsewhere

**Review by Hugoo — APPROVED** [review]

**Pull request merged**